### ISSUE TITLE
Incorrect boilerplate for EPI to T1 registration

### ISSUE THREAD

**Issue by mnarayan** [poldracklab/fmriprep#1323]
Despite using the flag `--bold2t1w-dof 6 `, the boilerplate reports `nine degrees of freedom`

```
The BOLD reference was then co-registered to the T1w reference using
`flirt` [FSL 5.0.9, @flirt] with the boundary-based registration [@bbr]
cost-function.
Co-registration was configured with nine degrees of freedom to account
for distortions remaining in the BOLD reference.
```

**Comment by tsalo** [comment]
Was this fixed by #1701?

**Comment by effigies** [comment]
Yup, think so.
